Sources/Kitura/Kitura.swift

```diff
@@ -300,3 +308,30 @@ public class Kitura {
     internal private(set) static var httpServersAndUnixSocketPaths = [(server: HTTPServer, socketPath: String)]()
     internal private(set) static var fastCGIServersAndPorts = [(server: FastCGIServer, port: Port, address: String?)]()
 }
+
+// MARK: ServerOptions
+
```

Comment:
Update usage example

Reply:
Also need to test that this does not interfere with WebSockets / upgraded connections.